config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+# knocking URL needs to be customized in apache-pass.local
```

Comment:
If it NEEDS to be, then I'm not sure we should be still defining it in the filter's `[Init]` section, please correct me if I'm missing something. Otherwise, I'm not sure we can leave it undefined and have the tests pass?